doc/BACKUP.md

```diff
@@ -82,6 +82,92 @@ any in-channel funds.
 To recover in-channel funds, you need to use one or more of the other
 backup strategies below.
 
+## SQLITE3 `--wallet` `:` And Remote NFS Mount
```

Comment:
```suggestion
## SQLITE3 `--wallet` `{from}:{to}` And Remote NFS Mount
```

IMO this convention would be more understandable for the casual user?

Reply:
Maybe:

> ```markdown
> ## SQLITE3 `--wallet=${main}:${backup}` And Remote NFS Mount
> ```

Reply:
concept ack!